Assets/_Functionalities/Wms/Scripts/Legend.cs

```diff
@@ -0,0 +1,56 @@
+using Netherlands3D.Twin.Layers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+namespace Netherlands3D.Twin
```

Comment:
kan dit in `namespace Netherlands3D.Functionalities.Wms` dan is het alvast correct voor de scripts herordening